src/Files.App/Helpers/ServiceRegistration/AppLifecycleHelper.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Files.App.Services.DateTimeFormatter;
+using Files.App.Services.Settings;
+using Files.App.Storage.FtpStorage;
+using Files.App.Storage.NativeStorage;
+using Files.App.ViewModels.Settings;
+using Files.Core.Services.SizeProvider;
+using Files.Core.Storage;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.UI.Xaml;
+using System.IO;
+using Windows.Storage;
+
+namespace Files.App.Helpers.ServiceRegistration
+{
+	public class AppLifecycleHelper
```

Comment:
This class could be made static

Reply:
And remove ServiceRegistration. This can be moved to Helpers.UI but without namespace changes.